### PR TITLE
Kql magic updates

### PR DESCRIPTION
**Pull request by gupta1**
This PR fixes #
- Updates KqlMagic latest version.
- Adds KqlMagic environment settings to improve performance.
- Removes '--force reinstall' param for all pip installs/upgrades, defaults to "--user" param

### CODE REVIEW

File: extensions/notebook/src/jupyter/jupyterSessionManager.ts [hunk 1]
```diff
@@ -351,6 +351,9 @@ export class JupyterSession implements nb.ISession {
 				}
 			}
 
+			// Enable silent mode for perf improvement of reload command.
```
Comment (chlafreniere) [inline extensions/notebook/src/jupyter/jupyterSessionManager.ts]: does this fix the cmd window popping up, or is that something different?

File: extensions/notebook/src/jupyter/jupyterServerInstallation.ts [hunk 2]
```diff
@@ -588,9 +588,7 @@ export class JupyterServerInstallation implements IJupyterServerInstallation {
 
 		let versionSpecifier = useMinVersion ? '>=' : '==';
 		let packagesStr = packages.map(pkg => `"${pkg.name}${versionSpecifier}${pkg.version}"`).join(' ');
-		// Force reinstall in case some dependencies are split across multiple locations
-		let cmdOptions = this._usingExistingPython ? '--user --force-reinstall' : '--force-reinstall';
-		let cmd = `"${this.pythonExecutable}" -m pip install ${cmdOptions} ${packagesStr} --extra-index-url https://prose-python-packages.azurewebsites.net`;
+		let cmd = `"${this.pythonExecutable}" -m pip install --user ${packagesStr} --extra-index-url https://prose-python-packages.azurewebsites.net`;
```
Comment (corivera) [inline extensions/notebook/src/jupyter/jupyterServerInstallation.ts]: Can you also remove force-reinstall from the conda commands?

File: extensions/notebook/src/jupyter/jupyterSessionManager.ts (hunk 1 above)
Comment (gupta1) [reply extensions/notebook/src/jupyter/jupyterSessionManager.ts]: I don't see that locally. But that's something different. If it persists, we need to check with Michael.

File: extensions/notebook/src/jupyter/jupyterServerInstallation.ts (hunk 2 above)
Comment (gupta1) [reply extensions/notebook/src/jupyter/jupyterServerInstallation.ts]: Sure, I have removed it.

**Pull request merged**